src/transforms/globalCssToCssModule/globalCssToCssModule.ts

```diff
@@ -122,6 +131,7 @@ export async function globalCssToCssModule(options: GlobalCssToCssModuleOptions)
     const codemodResults = await Promise.all(codemodResultPromises)
 
     if (shouldWriteFiles) {
+        signale.info('Persisting codemod changes to the filesystem...')
```

Comment:
Can you add a  `signale.info('Persisting codemod changes completed')` when the process is done?